.github/workflows/ci.yml

```
@@ -166,16 +167,17 @@ jobs:
     # MSVC, but for now, this action works:
     - name: Prepare compiler environment for Windows 🐍 2.7
       if: matrix.python == 2.7 && runner.os == 'Windows'
-      uses: ilammy/msvc-dev-cmd@v1
+      uses: ilammy/msvc-dev-cmd@v1.4.1
```

```suggestion
      uses: ilammy/msvc-dev-cmd@v1
```

The v1 tag was [fixed upstream](https://github.com/ilammy/msvc-dev-cmd/issues/16). Thanks @ilammy!